packages/warehouses/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@lightdash/warehouses",
```

Comment:
I'd use `warehouses`, @lightdash is only needed if we are planning to release this package to NPM under @lightdash scope

Reply:
I really liked the namespace for the `cli` because it makes the `package.json` look cleaner! And it's much more obvious in files when you see the import that it comes from the mono repo and not a 3rd party package.

 Is there any downside?

Reply:
Not really, I find it a bit long to type `yarn workspace @lightdas/xxx ` but I see your point